### PR TITLE
💅 customize cookies

### PR DESCRIPTION
**Pull request by roiLeo**
In Accordance with GDPR, user has now control over tracking cookies. 

- ref #9350 
- ref #9152 
- remove/replace old `vue-gtag` implementation

### Screenshots

Screenshot 2024-02-14 at 19-37-45 KodaDot - NFT Market Explorer
Screenshot 2024-02-14 at 19-25-10 KodaDot - NFT Market Explorer
Screenshot 2024-02-14 at 19-26-30 KodaDot - NFT Market Explorer

**Accept**
Capture d’écran 2024-02-14 à 7 17 12 PM

**Decline**
Capture d’écran 2024-02-14 à 7 16 46 PM

### What's missing
- [ ] small UI changes
- [ ] clarity setting
- [ ] double check text
- [ ] locales/translation

### CODE REVIEW

**Comment by prury** [comment]
wheres my data-testid 😢

**Comment by yangwao** [comment]
Hey @roiLeo thanks for PR

I don't see the text "Being on our site, you agree with our cookie policy", can you adjust please?

Is it default on?

**Comment by roiLeo** [comment]
> wheres my data-testid 😢 (i accept it on tests because it sometimes gets in the way of some elements I'm trying to click on page)

✅ fixed using page selector instead of data-id

> can't disable/enable cookie control, is it supposed to be like that?

this is excepted see title "Necessary cookies", some cookies are needed for proper operation like theme color, if the cookie banner has already been accepted

> mobile
✅ fixed

> I don't see the text "Being on our site, you agree with our cookie policy", can you adjust please?

Screenshot 2024-02-15 at 10-01-34 KodaDot - NFT Market Explorer

> Is it default on?

yes, default ON, even on page load

**Comment by prury** [comment]
> > wheres my data-testid 😢 (i accept it on tests because it sometimes gets in the way of some elements I'm trying to click on page)
> 
> ✅ fixed using page selector instead of data-id
> 
> > can't disable/enable cookie control, is it supposed to be like that?
> 
> this is excepted see title "Necessary cookies", some cookies are needed for proper operation like theme color, if the cookie banner has already been accepted
> 
> > mobile
> > ✅ fixed


thank you

**Review by preschian — APPROVED** [review]

**Comment by prury** [comment]
👀

File: components/CookieBanner.vue [hunk 1]
```diff
@@ -1,66 +1,106 @@
...
+
+/* Custom Cookie #Modal */
+.cookieControl__ModalContent {
+  @apply text-text-color bg-background-color border border-border-color shadow-primary items-center overflow-y-clip;
+}
+.cookieControl__ModalContent :not(button) {
+  @apply text-text-color;
+}
+button.cookieControl__ModalClose {
+  font-family: 'Font Awesome 6 Sharp';
```
Comment (Jarsen136) [inline components/CookieBanner.vue:64]: Could we remove the `font family`?
image

Comment (roiLeo) [reply components/CookieBanner.vue]: ✔️

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks @roiLeo for finishing this one.

**Pull request merged**